eslint-config-lib/index.js

```diff
@@ -22,8 +22,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 module.exports = {
   extends: [
     "@inrupt/eslint-config-base",
-    "plugin:@typescript-eslint/recommended",
-    "prettier/@typescript-eslint"
+    "plugin:@typescript-eslint/recommended"
```

Comment:
Oh, wait, I see you've done #18 here instead, I can close that if we're ready here — though do we want to upgrade to eslint 8.4.1?